docs-en/chapter_preface/suggestions.md

```diff
@@ -2,186 +2,186 @@
 
 !!! tip
 
-    For the best reading experience, it is recommended that you read through this section.
+    It is recommended that you go through this section to get the best reading experience.
 
-## Conventions Of Style
+## Conventions Of Writing Style
 
-- Those labeled `*` after the title are optional chapters with relatively difficult content. If you have limited time, it is advisable to skip them.
-- Proper nouns and words and phrases with specific meanings are marked with `"double quotes"` to avoid ambiguity.
-- Important proper nouns and their English translations are marked with `" "` in parentheses, e.g. `"array array"` . It is recommended to memorize them for reading the literature.
+- Chapters marked with an asterisk (`*``) after the title are supplementary and more challenging. If you have limited time, it is advisable to skip them.
```

Comment:
Chapters marked with an asterisk `*` in their titles are optional and more challenging. If you are short on time, it is advisable to skip them.